stock_sim/modules/dbWriter.py

```diff
@@ -1,7 +1,6 @@
 import mysql.connector
 import os, sys
 from datetime import datetime
-
```

Comment:
For future, i woul leave a space between you imported libraries and your classs

Reply:
edited